tests/base.py

```diff
@@ -279,7 +293,11 @@ def cleanup(self):
                 except AttributeError:
                     continue
 
-        if self.remove_legend:
+        if self.remove_labels:
+            self.ax.set_xlabel("")
+            self.ax.set_ylabel("")
```

Comment:
Just note that this solves the xlabel, ylabel issue that has popped up in mtpl 3.x